docs/user/guides/access/index.md

```diff
@@ -1,5 +1,9 @@
 ---
-title: Access
+title: Accessing your DAppNode
 ---
+---
+This section covers different ways of accessing your DAppNode.  
+
+We recommend that you start setting up basic access via [WiFi](wifi) (recommended!) or the [local proxy method](local) (this only offers a restricted DAppNode UI). 
 
-Here you can learn how to access your dappnode and how to set up the different kind of access. It's not necessary to set up all the access. From DAppNode, we recommend setting up the basic access via local-proxy, wifi. And in the future try to set up VPN access in case you want.
+> :warning: Once you have that all set up and you realize want to access your DAppNode on the go, we recommend that you take a look at the [VPN](vpn) guides.
```

Comment:
emphasize this a bit more somehow make it **bolder or increase font size for the line,**  TL;DR my explanation of why
 ``` VPN access is definitely best in almost all use cases, wifi is a security risk, local proxy is too and remains somewhat buggy (for me it's always buggy worked like 10 times since it was implemented and never worked for more than a couple hours if that, users often have issues with it too given support requests in discord)  we just need to streamline the VPN setup process and then document it super well, some of which is  already done, will open an issue for that.  ```